x/auth/migrations/legacytx/stdsignmsg.go

```diff
@@ -21,7 +21,9 @@ type StdSignMsg struct {
 
 // get message bytes
 func (msg StdSignMsg) Bytes() []byte {
```

Comment:
This function appears to unused but is public, can it be removed to fully deprecate StdSignBytes?